src/js/brim/record.js

```diff
@@ -14,9 +22,14 @@ export default function record(data: RecordData): $Record {
     data() {
       return data
     },
-    field(name: string) {
+    get(name: string) {
```

Comment:
maybe `getField` ?

Reply:
I had that at first but later felt that it is clear enough when I see it in context: `record.get("_path")`.